pkgs/development/libraries/boost/python.nix

```diff
@@ -0,0 +1,100 @@
+{
+  lib,
+  stdenv,
+  fetchpatch2,
+  boost,
+  boost-build,
+  libxcrypt,
+  python,
+  fixDarwinDylibNames,
+  toolset ? /**/ if stdenv.cc.isClang  then "clang"
+            else if stdenv.cc.isGNU    then "gcc"
+            else null,
+}:
+let
+  b2Args = lib.concatStringsSep " " ([
+    "--with-python"
+    "--user-config=user-config.jam"
+    "--includedir=${lib.getInclude boost}/include"
+    "--libdir=$out/lib"
+    "-j$NIX_BUILD_CORES"
+    "--layout=system"
+    "variant=release"
+    "debug-symbols=off"
+    "threading=multi"
+    "link=${if stdenv.hostPlatform.isStatic then "static" else "shared"}"
+    "runtime-link=${if stdenv.hostPlatform.isStatic then "static" else "shared"}"
+  ]
+    ++ lib.optional (toolset != null) "toolset=${toolset}"
+  );
+in
+stdenv.mkDerivation (finalAttrs: {
+  pname = "boost.python";
+  inherit (boost) version src meta;
+  outputs = [ "out" "dev" ];
+
+  patches = [
+    (fetchpatch2 {
```

Comment:
Needs conditionalizing like in the up‐to‐date Boost derivation (and removing from `generic.nix`, of course).